package.json

```diff
@@ -170,7 +174,7 @@
     "stylelint-order": "5.0.0",
     "stylelint-value-no-unknown-custom-properties": "4.0.0",
     "tsc-files": "1.1.4",
-    "typescript": "5.0.4",
+    "typescript": "5.2.2",
```

Comment:
Is this update needed for this PR?

Reply:
No, I'm not sure how that happened. I'll restore it back to the original version